php_remote_error_monitor.h

```diff
@@ -48,7 +48,13 @@
     #define PHP_REMOTE_ERROR_MONITOR_API
   #endif
 
-  #define REMOTE_ERROR_MONITOR_E_ALL (E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_WARNING | E_STRICT)
+  /* Saving this specification, as we may want to return to it when our
+   * error reporting mechanism isn't causing platform stability issues.
+
+    #define REMOTE_ERROR_MONITOR_E_ALL (E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_WARNING | E_STRICT)
+   */
+
+  #define REMOTE_ERROR_MONITOR_E_ALL (E_ERROR & E_PARSE & E_CORE_ERROR & E_COMPILE_ERROR & E_USER_ERROR)
```

Comment:
Whups, good call.  Fixed.

We can use something like the [trigger_error()](https://www.php.net/manual/en/function.trigger-error.php) to test the different types and ensure the new code is working the way we  intend.

Reply:
If you know how to make this configurable from php.ini, I'm open to that change, but I've had trouble getting module-specific settings to work.  We should not use the global setting because we should control what goes to the _dashboard_ independently of what goes to the _log file_.  

It is not configured from php.ini now, so I don't think we should hold up the fix to an issue causing platform instability when what's currently in production isn't any better.

Reply:
> This can be a follow-on issue, but we should address.

Can you capture in a JIRA ticket or Github issue so that this is not lost?